lib/Sabberworm/CSS/Parser.php

```diff
@@ -520,6 +524,17 @@ private function parseURLValue() {
 		return $oResult;
 	}
 
+	private function parseCalcValue() {
+		$func = $this->comes('calc', true) ? 'calc' : '-webkit-calc';
+		$this->consume($func);
+		$this->consumeWhiteSpace();
+		$this->consume('(');
+		$aArguments = $this->parseValue(array('+', '-', '*', '/', ' '), '\Sabberworm\CSS\Value\CalcRuleValueList');
```

Comment:
Parsing the arguments using `parseValue` probably means an expression like `calc(50% -8px)` would be treated as `calc(50% - 8px)`, not as an error (like it should).

Reply:
Am I right in assuming that this does not support parenthesized subexpressions like `calc( ( 100px / 2) / 2)`?

Reply:
Correct for both. Should be fixed now :)